core/src/test/java/io/questdb/griffin/AlterTableDropPartitionTest.java

```diff
@@ -312,10 +382,10 @@ public void testDropPartitionWrongSeparator() throws Exception {
     @Test
     public void testDropPartitionsByDayUsingWhereClause() throws Exception {
         assertMemoryLeak(() -> {
-                    createX("DAY", 720000000);
+            createX("DAY", 720000000);
```

Comment:
nit: this looks like some broken formatting considering the indentation on L390 and below.

Reply:
How to fix it? Is a setting?

Reply:
That's a good question. It seems that IDEA has some bugs around its formatter. For instance, I tried to get rid of the extra trailing newlines in `DataFrameRecordCursor` today and IDEA auto-formatted the file in a totally broken way.

In the worst case, you could try formatting this method manually, but maybe adding/removing newlines may force IDEA to behave as expected.